policy-controller.md

```diff
@@ -0,0 +1,65 @@
+---
+title: policy-controller
+category: server-app
+tags: go kubernetes
+permalink: /policy-controller
+releasePolicyLink: https://github.com/sigstore/policy-controller#support-policy
+releaseDateColumn: true
+eolColumn: false
+
+identifiers:
+  - purl: pkg:github/sigstore/policy-controller
+  - repology: sigstore
```

Comment:
This seems like the package for sigstore-python in homebrew only at this point:
```suggestion
```
https://repology.org/project/sigstore/information